app/bundles/CampaignBundle/Command/TriggerCampaignCommand.php

```diff
@@ -167,6 +168,34 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $threadId      = $input->getOption('thread-id');
         $maxThreads    = $input->getOption('max-threads');
 
+        if (is_numeric($id)) {
```

Comment:
Shouldn't !is_numberic condition?
Based on code make more sense
Other places as well

Reply:
@kuzmany The reason for this check is that only numeric values will be casted to `int` properly. Non numeric values should raise a type error, or have an error message in `else`.

Reply:
@escopecz your opinion?

Reply:
It depends wether you intend to use the cast to int as a user value sanitization. How will this work with

`bin/console mautic:campaigns:trigger --campaign-id=unicorn`

Is the code fine with working with strings? Is retyping the string to an int valid? Quick test:
```
php > var_dump((int) 'unicorn');
int(0)
php > var_dump((int) '100unicorn');
int(100)
```
Ideally, it should guide the user and reject invalid values. But that would deserve its own PR.

I understand what @biozshock meant to accomplish here. This way it won't convert `null` values to `0`. But I understand what @kuzmany means too. There are other values than `int` and `null`.

Reply:
I'm suggesting this: https://github.com/mautic/mautic/pull/13110/files#r1453534132